RELEASE.md

```diff
@@ -10,7 +10,9 @@ Our goal is to provide a new minor release every 4 weeks. This is a new process
 |----------------|--------------------------------------------|---------------------------------------------|
 | v0.1.0         | 2019-07-31                                 | Chris Marchbanks (GitHub: @csmarchbanks)    |
 | v0.2.0         | 2019-08-28                                 | Goutham Veeramachaneni (Github: @gouthamve) |
-| v0.3.0         | 2019-10-09                                 | **searching for volunteer**                 |
+| v0.3.0         | 2019-10-09                                 | Bryan Boreham (@bboreham)                   |
+| v0.4.0         | 2019-11-13                                 | Tom Wilkie (@tomwilkie)                     |
```

Comment:
Currently its counting 5 weeks.
```suggestion
| v0.4.0         | 2019-11-06                                 | Tom Wilkie (@tomwilkie)                     |
```

Reply:
I figured it would be better to do it the week after Promcon EU.

Reply:
Yup, that makes sense. Didn't think about that!